pkg/bundle/internal/truststore/types_test.go

```diff
@@ -146,3 +143,60 @@ func Test_certAlias(t *testing.T) {
 		t.Fatalf("expected alias to be %q but got %q", expectedAlias, alias)
 	}
 }
+
+func Test_manual_read_JKS(t *testing.T) {
+	t.Skip("these tests require installation of Java to run")
```

Comment:
suggestion: maybe we could look in $PATH for keytool, and only skip this test if keytool isn't found?